CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+### v0.3.1
+
+v0.3.1 is the first release with a changelog :-)
+
+There are also a number of new features in this release, including [groups](README.md#groups), an [`off` strategy](README.md#configuration), and [reboot windows](README.md#reboot-windows).
+
+Changes since v0.3.0:
+- addition of a "groups" feature (#70), facilitating partitioned co-ordinating of reboots
```

Comment:
maybe list the new features in level 2 under one level 1 item
- new features
  * groups
  * "off" strategy
  * reboot windows